src/amber/environment/settings.cr

```diff
@@ -1,9 +1,8 @@
-require "logger"
 require "yaml"
 
 module Amber::Environment
   class Settings
-    property logger : ::Logger = Logger.new(STDOUT)
+    property logger : Logger = Environment::Logger.new(STDOUT)
```

Comment:
Weren't we going to remove this from settings since we don't have a way to configure it through env.ymls anymore?

Reply:
I didn't remove since people can still change it via `config/application.cr`, we can reconsider in a separate PR.

Reply:
Ok.

Reply:
We do probably need to think of some way to at least configure log level via env.yml's. Since people probably want different things logged in different environments.

Reply:
It might be good to do it in this PR since it relates to logging and configuring logging level. This next release will contain all of our changes to settings to it would make sense to include that unless it's a huge rework which it doesn't seem like.